examples/community.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang = "en">
+<head>
+  <!-- build:css ../css/bootstrap.min.css -->
+  <link href="../html/css/bootstrap.min.css" rel="stylesheet">
+  <!-- /build -->
+  <!-- build:css ../css/style.css -->
+  <link href="../html/css/style.css" rel="stylesheet">
+  <!-- /build -->
+  <link href="common/css/style.css" rel="stylesheet">
+</head>
+<body>
+<div class="container">
+  <div class="page-header">
+    <h1>Community Examples</h1>
+    <p class="lead">
+    Here are a couple examples of what people are building with Rocky.js. Open a pull request to add yours!
```

Comment:
Strictly speaking this is just *one* example for now.

Also, what do you think about linking "open a pull request" to https://github.com/pebble/rockyjs/compare ?